tests/integration/ceph_helm_test.go

```diff
@@ -59,17 +59,19 @@ func TestCephHelmSuite(t *testing.T) {
 
 type HelmSuite struct {
 	suite.Suite
-	helper    *clients.TestClient
-	kh        *utils.K8sHelper
-	op        *TestCluster
-	namespace string
+	helper          *clients.TestClient
+	kh              *utils.K8sHelper
+	op              *TestCluster
+	namespace       string
+	rookCephCleanup bool
 }
 
 func (hs *HelmSuite) SetupSuite() {
 	hs.namespace = "helm-ns"
 	mons := 1
 	rbdMirrorWorkers := 1
-	hs.op, hs.kh = StartTestCluster(hs.T, helmMinimalTestVersion, hs.namespace, "bluestore", true, false, "", mons, rbdMirrorWorkers, installer.VersionMaster, installer.NautilusVersion)
+	rookCephCleanup := true
```

Comment:
So we are only using the cleanup policy for the Helm suite? Why is this?

Reply:
Good question. The initial intention was that we wanted to get integration test coverage. But now I realize that after we get the device cleanup implemented with this, we should remove our wipe devices job and just rely on this for the test suites to cleanup.